rllib/examples/quadx_waypoints.py

```diff
@@ -0,0 +1,125 @@
+"""Example using the PyFlyt Gymnasium environment to train a UAV to reach waypoints.
```

Comment:
Awesome stuff! Thanks for this PR @peterghaddad !

Looks good to me. After merging this, we should add it to the examples docs (which we'll also have to clean up a bit, still).

Reply:
@sven1977 thank you for reviewing and your feedback. I addressed your comments.

I think that's a great idea. I'll write up an issue #45586